server/plugin.go

```diff
@@ -296,13 +300,15 @@ func (p *Plugin) handleComplete(w http.ResponseWriter, r *http.Request) {
 		p.handleErrorWithCode(w, http.StatusBadRequest, "Unable to decode JSON", err)
 		return
 	}
+	p.API.LogError("user ID: " + userID + "completeRequest.ID: " + completeRequest.ID)
```

Comment:
I understand this is some debug log here. Can it be removed?

Reply:
thank you for noticing